service/templates/projects.html

```diff
@@ -765,6 +765,28 @@ <h3>Patch Aliases</h3>
           </div>
         </div>
 
+        <!-- Patch Trigger Aliases -->
+        <div class="variables">
```

Comment:
Is the work to make patch triggers do something going to be done in another ticket? In which case this should be hidden for now, so that current project admins don't see it and expect the feature to be complete.

Reply:
done

Reply:
The aliases in-and-of themselves won't cause anything to happen. They are available for users to specify when they create a patch, coming to a theater near you.